Sources/MongoSwift/BSON/CodableNumber.swift

```diff
@@ -140,11 +140,6 @@ extension UInt: CodableNumber {
 extension Double: CodableNumber {
     internal init?(from value: BSONValue) {
         switch value {
-        case let v as Int:
```

Comment:
the compiler now points out that a cast from `BSONValue` to `Int` always fails since `Int` no longer conforms to `BSONValue`.